### DISCUSSION TITLE
Welcome to Lithnet Access Manager Discussion

### DISCUSSION THREAD

**Discussion by ryannewington** [lithnet/access-manager#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Please do not use this discussion forum to raise bugs. Use the issues function for that

**Comment by sylwester-majcher** [comment]
Hi, we will be testing last release of Access Manager. Thanks a lot for this product it looks amazing.
From my end I am really interesting of JIT feature!
If we mean about new idea and because this is Access Manager...why not add some feature to manage USB access for removable drive for example? I dont thint about managing guides of devices. Some of simple solution to turn on or turn off usb for removeble drives. I know there is GPO entries we can use but when computers are on the vpn and some times you need to change computer groups assosiation, this is nightmare.
So simply my idea: in freeware license: poor turn on/off management for usb remov drives
In enterprise: more granular as managing of guide or model or producer etc
